services/admin/handlers/resetData.js

```diff
@@ -90,6 +143,37 @@ export const main = async (event) => {
     })
   );
 
+  await Promise.all(
+    snapshotIds.map(async (id) => {
+      console.log("Checking for id: ", id);
+      const qParams = {
+        TableName: process.env.oneMacTableName,
+        KeyConditionExpression: "pk = :inPk",
+        ExpressionAttributeValues: {
+          ":inPk": id,
+        },
+        ProjectionExpression: "pk,sk",
+      };
+      try {
+        const results = await dynamoDb.query(qParams).promise();
+        console.log("Found these results in One Table: ", results);
+        for (const item of results.Items) {
+          const [, eventTimestamp] = item.sk.split("#");
+          if (eventTimestamp > 1676384054014)
```

Comment:
I feel like i remember you saying something about using Feb14th as a cutoff date but I dont remember the reason! Assume it has to do with only wanting to delete test data created by automation that has that later timestamp but to keep some base records?